db-experiment/Vagrantfile

```diff
@@ -0,0 +1,28 @@
+# -*- mode: ruby -*-
+
+box = "generic/ubuntu1804"
+ram = "65536"
+hostname = "wpia-db-experiment"
+mac = "00:15:5d:1a:84:10"
+cpus = 2
```

Comment:
Looks like're already 8 cpus over budget over the machine! So I guess 2 is the min here, but something that we might look at soon

Reply:
What does this core count add up to?

We have 2 24 core processors multithreaded to 48 each. So are we already using 34/96?

Reply:
2 12 core processors, 24 real cores, 48 hyperthreaded I think (based on a google)

Reply:
Two of these beasts - https://ark.intel.com/content/www/us/en/ark/products/192481/intel-xeon-platinum-8268-processor-35-75m-cache-2-90-ghz.html

48 real cores, 96 hyperthreaded

Reply:
In any case, with our server-type VMs, total CPU usage of all the VMs currently running is 1% (and the windows scheduler isn't allocating particular cores to particular VMs I think?) . I wonder what it looks like when the VMs are more stressed?